Hortali-App/iOS-App/Screens/Favorite/View/FavoriteView.swift

```diff
@@ -12,13 +12,19 @@ class FavoriteView: MainView {
     // Views
     
     /// Collection de alimentos
-    private let foodGroup = CollectionGroup()
+    private let foodGroup = CollectionGroup(emptyViewType: .food)
     
     /// Collection das hortas
-    private let gardenGroup = CollectionGroup()
+    private let gardenGroup = CollectionGroup(emptyViewType: .garden)
     
-    
-    // Outros
+    /// instancia da tela de empty view
+    private let emptyView = {
+        let view = EmptyView()
+        view.dynamicLabel.text = "Você ainda não tem\n nenhum favorito."
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Se vc já está criando um componente do zero, essa linha não faz sentido estar na hora de instanciar o componente, principalmente que toda vez que for usar esse componente vai ter que fazer essa configuração.

Coloque essa linha no próprio inicializador do componente, ou seja: coloque dentro do `init()` da `EmptyView()`